spec/java_integration/fixtures/SubclassOfClassWithSimpleMethod.java

```diff
@@ -0,0 +1,12 @@
+package java_integration.fixtures;
+
+/**
+ * Reopened Java subclasses should super normally into the parent class, not using reified subclass logic.
+ *
+ * See jruby/jruby#6968
+ */
+public class SubclassOfClassWithSimpleMethod extends ClassWithSimpleMethod {
+    public String foo(String s) {
+        return "foo" + s;
+    }
```

Comment:
Is this actually a valid test for the issue? My original test case for #6968 does not override `foo()` in the subclass.

Reply:
Aha, great catch! The fix does pass with this override removed, but I copy/pasted the wrong code into the body of the subclass. The subclass is fixed in c5bb622a3d.